connect/runtime/src/test/java/org/apache/kafka/connect/integration/RebalanceSourceConnectorsIntegrationTest.java

```diff
@@ -62,9 +62,10 @@
     private static final Logger log = LoggerFactory.getLogger(RebalanceSourceConnectorsIntegrationTest.class);
 
     private static final int NUM_TOPIC_PARTITIONS = 3;
-    private static final long CONNECTOR_SETUP_DURATION_MS = TimeUnit.SECONDS.toMillis(30);
-    private static final long WORKER_SETUP_DURATION_MS = TimeUnit.SECONDS.toMillis(60);
+    private static final long CONNECTOR_SETUP_DURATION_MS = TimeUnit.SECONDS.toMillis(90);
+    private static final long WORKER_SETUP_DURATION_MS = TimeUnit.SECONDS.toMillis(90);
```

Comment:
Part 1: I increased the durations from 30 and 60 seconds to 90 seconds. Quite a few of the test failures timed out while waiting for Connect to start or the connector to be started. I suspect the Jenkins machines are slower than my dev machine, too.